app/shared/availability-view/_availability-view.scss

```diff
@@ -254,6 +254,9 @@
   .reservation-slot {
     height: $slot-height;
     position: relative;
+    margin: 0;
+    padding: 0;
+    border: none;
```

Comment:
Are these styles enough to make the <button> element to look like the link did before?

Reply:
Yes, because the classNames were passed as props to the deleted component, so all the previous styles were applied to the `button` element.

Reply:
What I meant was did you make sure that the browsers default button styles don't appear?

Reply:
Yes, at least in Chrome, let me double check in other browsers.

Reply:
@rBoost it seems to be fine.

Reply:
👍